chef/cookbooks/ohai/files/default/plugins/linux/s390x.rb

```diff
@@ -0,0 +1,40 @@
+provides "s390x"
+
+if File.exist? "/proc/sysinfo"
```

Comment:
I guess this file something s390 specific? Or do we need some other way to ensure this plugin is only run on s390 machines?